### DISCUSSION TITLE
I am having unknown key value error

### DISCUSSION THREAD

**Discussion by Aswin2203** [IETF-Hackathon/pqc-certificates#144] [General]
My Query is about is there any library in python that can generate a X509 certificate using the Post Quantum Cryptography [Open Quantum Safe] Algorithms Key pair [like dilithium] Or is there any other way to generate a X509 certificate using PQC key pairs. It would be so much helpful if you can provide me a solution.

As if I generate a certificate with dilithium key pair using cryptography library, I get this error
ValueError: Unknown key type: 1.3.6.1.4.1.2.267.12.6.5

**Comment by johngray-dev** [comment]
I think Corey would know the answer to your question, and he is planning to be at the hackathon next weekend.  Are you planning to be there as well?  If not, let me know your e-mail and you can join as a remote participant.
